src/api/model.js

```diff
@@ -34,4 +34,16 @@ app.post('/', (req, res) => {
         });
 });
 
+app.delete('/', (req, res) => {
+    addAction('DELETE_MODEL', {
+        modelAddress: req.query.modelAddress
+    }).then(() => {
+        res.successJson({
+            models: ModelStore.getModels()
```

Comment:
I think have two REST calls would not be a big deal since we don't expect scalability to be a problem (e.g. having 1 million models that need to transferred).
Also, I don't think we would need to assume the user's next action. In other words, I don't think we should assume that after the user deletes a model, they navigate to a page that needs a list of all models.

In brief, I think this ought to be changed to respond with the deleted item, however I do not feel strongly about it.